src/com/facebook/buck/rules/BUCK

```diff
@@ -47,6 +47,11 @@ java_immutables_library(
     ],
 )
 
+genrule(name = "foo",
```

Comment:
changes to this build file should be part of an integration test - not production code ;)

Reply:
If you want an existing test to model, here's where most of the genrule query tests live so far: https://github.com/facebook/buck/blob/master/test/com/facebook/buck/shell/GenruleDescriptionIntegrationTest.java and here's where their buck files are: https://github.com/facebook/buck/blob/master/test/com/facebook/buck/shell/testdata/genrule_classpath_filtering/BUCK.fixture

Reply:
Ha! Ooops. :) Rolled back to master's